### DISCUSSION TITLE
Unfinished issues/PRs

### DISCUSSION THREAD

**Discussion by Confectrician** [openhab/openhab-docs#2056] [Team Posts]
Hi all,

As someone may have recognised i am currently trying to put some effort in the docs repo structure.
We are talking about simplifying the repo look in openhab/openhab-docs#665 already, mainly to prevent contributors (and especially possible new contributors) from confusion.

I have just restructured our labels a bit (names and colors) to give them some more information and consistence in the future.

What i wanted to do additionally is talk/discuss about some of the issues/PRs with the unfinished label.
Some of them have bin inactive for months/years and unfortunately i doubt that there will be final progress in some time.
Also i think that open (longtime) inactive PRs could send a wrong signal like "a hey there's already a PR for jsondb, so i don't need to put effort in here".
I would like to close some or all of them, but wanted to get some opinions.
Mainly because those PRs were before i started helping out with the docs and i think they have been labeled with a reason some time ago.

We can reopen them anytime if activity gets back and of course if it makes sense and the problems haven't already been solved in another PR.

Wdyt?

BR
Jerome

**Comment by ThomDietrich** [comment]
Hey!

Thanks for your effort first of all!

Old issues are nothing new and totally normal. It's still okay to have them for reference or to be picked up later. We should only close some if their topic is irrelevant now, because it was solved or their vantage point disappeared. With that said: Feel free to go at it aggressively 😉 

Regarding PRs: Yes that is a bummer. I didn't want to close them. I always hoped the original authors would pick them up. As this is not the case one of us (you, see below) should pick up and complete them with as little effort as possible. 

Btw. As you know I am working on the final steps of my PhD. I am currently not checking GitHub notifications on a regular basis. Desperate times 😅 If anything comes up that you'd like my opinion on, don't hesitate to contact me via email!!
I'll be back to more-or-less normal in September!

Best! Thomas
